protocol-models/bin/generate-typescript-classes.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+git config --global --add safe.directory /airbyte-protocol
+
+ROOT_DIR=${ROOT_DIR:-$(git rev-parse --show-toplevel)}
+
+[ -z "$ROOT_DIR" ] && exit 1
+
+cd protocol-models/typescript
+npm install
```

Comment:
Should we maybe use `pnpm` to be aligned with the webapp as well as avoiding running into the same npm issues (non determinism, performance) than we did with the webapp?

Reply:
sold!